sqlserver-delta-plugins/src/e2e-test/java/io.cdap.plugin/BQValidation.java

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (c) 2023.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin;
+
+import com.google.cloud.bigquery.TableResult;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import org.junit.Assert;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Time;
+import java.sql.Timestamp;
+import java.sql.Types;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * BQValidation.
+ */
+public class BQValidation {
+
+  /**
+   * Extracts entire data from source and target tables.
+   *
+   * @param sourceTable table at the source side
+   * @param targetTable table at the sink side
+   * @return true if the values in source and target side are equal
+   */
+
+  public static boolean validateDBToBQRecordValues(String schema, String sourceTable, String targetTable)
+    throws SQLException, ClassNotFoundException, IOException, InterruptedException, ParseException {
+    List<Object> targetBigQueryRows = new ArrayList<>();
+    List<JsonObject> targetBigQueryResponse = new ArrayList<>();
```

Comment:
can we name them as bigQueryRows and bigQueryResponse

Reply:
done.